test/views/components/LateSubmissionPenaltySummaryCardSpec.scala

```diff
@@ -599,6 +669,27 @@ class LateSubmissionPenaltySummaryCardSpec extends SpecBase with ViewBehaviours
         docWithAppealedPointUnderTribunalRejected.select("dd").get(4).text() shouldBe "Appeal rejected by tax tribunal Read outcome message"
       }
     }
+
+    "given multiple penalty period in LSP" should{
+      implicit val doc: Document = asDocument(summaryCardHtml.apply(summaryCardModelWithMultiplePenaltyPeriodLSP))
+      "show message VAT return submitted earlier in multiple penalty period" in{
+        doc.select("p.govuk-body").text() shouldBe "The VAT Return due on 24 May 2021 was also submitted late. HMRC only applies 1 penalty for late submission in each month."
+      }
+    }
+
+    "given multiple penalty period in LSPP" should{
+      implicit val doc: Document = asDocument(summaryCardHtml.apply(summaryCardModelWithMultiplePenaltyPeriodLSPP))
+      "show message VAT return submitted earlier in multiple penalty period" in{
+        doc.select("p.govuk-body").text() shouldBe "The VAT Return due on 24 May 2021 was also submitted late. HMRC only applies 1 penalty for late submission in each month."
+      }
+    }
+
+    "given no multiple penalty period in LSP" should{
+      implicit val doc: Document = asDocument(summaryCardHtml.apply(summaryCardModelWithFinancialPointAboveThreshold))
+      "show message VAT return submitted earlier in multiple penalty period" in{
```

Comment:
```suggestion
      "show message VAT return submitted earlier in multiple penalty period" in {
```

Reply:
Also I'm not sure this test name makes sense - perhaps we could change this to be:
`"no message relating to multiple penalties in the same period should appear" in {`